### DISCUSSION TITLE
Is there a // minify-ignore type comment that I can place at the top of .js files that I don't want minified?

### DISCUSSION THREAD

**Discussion by wilvoss** [Josee9988/MinifyAll#139] [General]
Is there a // minify-ignore type comment that I can place at the top of .js files that I don't want minified?

**Comment by Josee9988** [comment]
It is not done yet.
The extension has a comment to ignore removing comments but not the entire code, you can check the [issue](https://github.com/Josee9988/MinifyAllCli/issues/65) or a [gif](https://gifs.com/gif/r2R4mB)

There is an open issue about ignoring certain files. I will try to do it as soon as I buy some free time, check [the issue here](https://github.com/Josee9988/MinifyAll/issues/131)

To sum up, this option is not yet available but I have plans to add it

Thanks for your comment!
